test/src/org/labkey/test/tests/targetedms/TargetedMSProteinGroupingTest.java

```diff
@@ -0,0 +1,60 @@
+package org.labkey.test.tests.targetedms;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.components.CustomizeView;
+import org.labkey.test.util.DataRegionTable;
+
+@Category({})
+@BaseWebDriverTest.ClassTimeout(minutes = 2)
+public class TargetedMSProteinGroupingTest extends TargetedMSTest
+{
+    private static final String SKY_FILE = "ProteinGroup.sky";
+
+    @BeforeClass
+    public static void initProject()
+    {
+        TargetedMSProteinGroupingTest init = (TargetedMSProteinGroupingTest) getCurrentTest();
+        init.doInit();
+    }
+
+    @Override
+    protected String getProjectName()
+    {
+        return getClass().getSimpleName() + " Project";
+    }
+
+    private void doInit()
+    {
+        setupFolder(FolderType.Experiment);
+        importData(SKY_FILE);
+    }
+
+    @Test
+    public void testProteinGrouping()
```

Comment:
this test looks good, but I think there is more functionality that got added like the last bullet in Changes in this PR list.